src/emulate.c

```diff
@@ -689,6 +695,7 @@ static uint32_t *csr_get_ptr(struct riscv_t *rv, uint32_t csr)
         return (uint32_t *) (&rv->csr_mtval);
     case CSR_MIP:
         return (uint32_t *) (&rv->csr_mip);
+     
```

Comment:
Minimize the necessary changes. That is, avoid such a blank line.

Reply:
Hi @jserv,
Resolved.
Thanks for your comment.